source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,321 @@ Sorts Builders
    :class: singlecol
 
 .. _sorts-builders:
+
+Overview
+--------
+
+In this guide, we show you how to use **builders** to specify
+**sort criteria** for your queries.
+
+Sort criteria are the rules MongoDB uses to sort your data. Some examples of sort criteria are:
+
+* Smallest number to largest number
+* Earliest to latest time of day
```

Comment:
For parallel structure, either:
"Smallest to largest number"
or
"Earliest time of day to latest time of day"

Reply:
Thank you, good catch. Fixed: https://github.com/mongodb/docs-java/pull/59/commits/a5a96aa86952d02dcb9bfb2255c098267a446364

I will try to keep an eye out non-parallel structure in my writing.